### ISSUE TITLE
Exporter: parameter type name bikeshedding

### ISSUE THREAD

**Issue by soininen** [Spine-project/Spine-Toolbox#1364]
@jkiviluo proposed some changes to Exporter's parameter type names to make filtering by type more user-friendly:

- `single_value` -> `scalar` (another proposal `constant` clashes with import mappings, strings are also scalars)
- `1d_map`, `2d_map` etc. -> `1d`, `2d` etc.
- No changes: `array`, `time_series`, `time_pattern`

**Comment by manuelma** [comment]
> 1d_map, 2d_map etc. -> 1d, 2d etc

What would be the purpose here? I feel it becomes a bit obscure.

**Comment by manuelma** [comment]
Can we add the word 'Importer' to the issue title as well? Because we need the consistency.

**Comment by jkiviluo** [comment]
> What would be the purpose here? I feel it becomes a bit obscure.

Yeah, I actually tend to agree. I've also been thinking that while currently `array` has only one dimension, we might want to later use it for multi-dimensional arrays (although I guess you could call them matrixes at that point). In any case, we should then also have names like `2x3_matrix` and having just `2d` would be confusing.

`Scalar` seems to be numeric. https://en.wikipedia.org/wiki/Scalar_(mathematics). So, I don't know how to stomach it. It might be that `single_value` is then preferable as it would be unambiguous.

**Comment by soininen** [comment]
To me it looks like we are after all happy with the current naming, am I correct? If no one disagrees I will soon close this issue as not-actually-an-issue.
